projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -1,26 +1,50 @@
-          <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
-          <ng-template #columnHeaderOutlet>
-            <ng-container *ngTemplateOutlet="col.goTableHead"></ng-container>
-          </ng-template>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr *ngFor="let item of localTableConfig.tableData" class="go-table__tr">
-        <td *ngFor="let col of columns" class="go-table__td" width="{{ col.width || 'auto' }}">
-          <ng-container *ngIf="!col.goTableCell; else columnCellOutlet">{{ col.getFieldData(item) }}</ng-container>
-          <ng-template #columnCellOutlet>
-            <ng-container *ngTemplateOutlet="col.goTableCell; context: { $implicit: item }"></ng-container>
-          </ng-template>
-        </td>
-      </tr>
-    </tbody>
-  </table>
+  <div class="go-table">
+    <table class="go-table__table" *ngIf="hasData(); else noDataTable">
+      <thead>
+        <tr>
+          <th *ngFor="let col of columns" class="go-table__th" [ngClass]="columnClasses(col.field)" (click)="toggleSort(col.field)">
+            <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
+            <ng-template #columnHeaderOutlet>
+              <ng-container *ngTemplateOutlet="col.goTableHead"></ng-container>
+            </ng-template>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let item of setDisplayData()" class="go-table__tr">
+          <td *ngFor="let col of columns" class="go-table__td" width="{{ col.width || 'auto' }}">
+            <ng-container *ngIf="!col.goTableCell; else columnCellOutlet">{{ col.getFieldData(item) }}</ng-container>
+            <ng-template #columnCellOutlet>
+              <ng-container *ngTemplateOutlet="col.goTableCell; context: { $implicit: item }"></ng-container>
+            </ng-template>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="go-table-toolbar go-table-toolbar--bottom" *ngIf="showPaging()">
+    <div class="go-table-page-size">
+      <select class="go-form__select" (change)="setPerPage($event)">
+        <option *ngFor="let size of localTableConfig.pageConfig.pageSizes"
+                value="{{ size }}">{{ size }}</option>
+      </select>
+      <label class="go-form__label go-table-toolbar__label">Page Size</label>
+    </div>
+    <div class="go-table-totals">
+      {{ outputResultsPerPage() }} /
+      {{ localTableConfig.totalCount | number : '1.0-0' }} Results
+    </div>
+    <div class="go-table-page-controls">
+      <go-icon class="go-table-page-controls__icon" [ngClass]="{ 'go-table-page-controls__icon--disabled': isFirstPage() }" (click)="setFirstPage()" icon="first_page"></go-icon>
+      <go-icon class="go-table-page-controls__icon" [ngClass]="{ 'go-table-page-controls__icon--disabled': isFirstPage() }" (click)="prevPage()" icon="chevron_left"></go-icon>
```

Comment:
These would be better if they were wrapped in a button. This would give you access to these controls via the keyboard.

Reply:
Great idea! Wish I would've thought about that first. I've added buttons here and it's a much better UX. 🥇  :+1: